README.md

```diff
@@ -411,15 +420,27 @@ All statements above will return false if used with `===`
 
 <br/><br/>
 
-## ![✔] 4.2 Detect code issues with a linter
+## ![✔] 4.2 Include 3 parts in each test name
+
+**TL;DR:** Make the test speak at the requirements level so it's self explanatory also to QA engineers and developers who are not familiar with the code internals. State in the test name what it being tested (unit under test), under what circumstances and what is the expected result
```

Comment:
I think there's a small typo in "State in the test name what it being tested..." -> "State in the test name what is being tested"

Reply:
@BrunoScheufler Yeah, good catch

Reply:
@BrunoScheufler Fixed, please approve